src/SessionSetup/include/SessionSetup/ConnectToTargetDialog.h

```diff
@@ -23,6 +23,14 @@ class ConnectToTargetDialog;  // IWYU pragma: keep
 }
 namespace orbit_session_setup {
 
+struct ConnectionTarget {
+  QString process_name_or_path;
+  QString instance_name_or_id;
+
+  ConnectionTarget(const QString& process_name_or_path, const QString& instance_name_or_id)
+      : process_name_or_path(process_name_or_path), instance_name_or_id(instance_name_or_id) {}
```

Comment:
Nit: This constructor is not needed, since aggregate initialization is doing the same for you.